packages/navigator/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Theia - Navigator Extension",
   "dependencies": {
     "@theia/core": "1.12.0",
+    "@theia/editor-preview": "1.12.0",
```

Comment:
Ideally we should not introduce new coupling between components, especially since nothing depends on `@theia/editor-preview` so applications currently have the flexibility of including it or not.

Reply:
I was able to remove this coupling in my latest commit 👍